libs/product/src/selectors/composite-product/composite-product.selectors.ts

```diff
@@ -274,26 +297,30 @@ const createCompositeProductSelectors = (): DaffCompositeProductMemoizedSelector
 			}
 
 			return daffSubtract(
-				selectCompositeProductMinPrice.projector(products, appliedOptionsEntities, { id: props.id }), 
-				selectCompositeProductMinDiscountedPrice.projector(products, appliedOptionsEntities, { id: props.id })
+				selectCompositeProductMinRequiredItemPrice.projector(products, appliedOptionsEntities, { id: props.id }), 
+				selectCompositeProductMinRequiredItemDiscountedPrice.projector(products, appliedOptionsEntities, { id: props.id })
+			) > 0 || daffSubtract(
+				selectCompositeProductMaxRequiredItemPrice.projector(products, appliedOptionsEntities, { id: props.id }), 
+				selectCompositeProductMaxRequiredItemDiscountedPrice.projector(products, appliedOptionsEntities, { id: props.id })
 			) > 0;
```

Comment:
I think these should be equality checks.

Reply:
agreed. changed